x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/config/ConfigTests.java

```diff
@@ -202,24 +202,15 @@ public void testNullTimeZone() {
     }
 
     public void testEmptyHistoField() {
-        HistoGroupConfig.Builder config = ConfigTestHelpers.getHisto();
-        config.setFields(null);
-        Exception e = expectThrows(IllegalArgumentException.class, config::build);
-        assertThat(e.getMessage(), equalTo("Parameter [fields] must have at least one value."));
-
-        config.setFields(Collections.emptyList());
-        e = expectThrows(IllegalArgumentException.class, config::build);
-        assertThat(e.getMessage(), equalTo("Parameter [fields] must have at least one value."));
+        final String[] fields = randomBoolean() ? new String[0] : null;
```

Comment:
should we test both all the time (no randomization) ?

Reply:
Ok